src/Controller/SponsorshipBenefit/Create.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\SponsorshipBenefit;
+
+use App\Form\SponsorshipBenefitType;
+use App\Service\DTO\SponsorshipBenefitData;
+use App\Service\Manager\SponsorshipBenefitManagerInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+use Twig_Environment;
+
+final class Create
+{
+    private $renderer;
+    private $sponsorshipBenefitManager;
+    private $formFactory;
+    private $router;
+
+    public function __construct(
+        Twig_Environment $renderer,
+        SponsorshipBenefitManagerInterface $sponsorshipBenefitManager,
+        FormFactoryInterface $formFactory,
+        RouterInterface $router
+    )
+    {
+        $this->renderer = $renderer;
+        $this->sponsorshipBenefitManager = $sponsorshipBenefitManager;
+        $this->formFactory = $formFactory;
+        $this->router = $router;
+    }
+
+    /**
+     * @param Request $request
+     * @return Response
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
+     */
+    public function handle(Request $request): Response
+    {
+        $sponsorshipBenefitData = new SponsorshipBenefitData();
+
+        $form = $this->formFactory->create(SponsorshipBenefitType::class, $sponsorshipBenefitData);
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $sponsorshipBenefitData->setPosition((int) $this->sponsorshipBenefitManager->getMaxPosition() + 1);
```

Comment:
Replace `+1` with a pre-incrementation. Also, why casting as an int? Shouldn't the return type of `$this->sponsorshipBenefitManager->getMaxPosition()` be an int already, hence removing the need for you to check the type?